source/data/globus_online_data_transfer.rst

```diff
@@ -109,3 +107,69 @@ Globus Connect Service is available on the following RDHPCS and partner clusters
    +===========+=====================+====================+======================+===============+
    | Hercules  | msuhpc2#Hercules    | /work, /work2      | Hercules DTN at MSU  | Anywhere      |
    +-----------+---------------------+--------------------+----------------------+---------------+
+
+NOAA RDHPCS Globus Endpoint Types
+=================================
+
+.. Note::
+
+  It is preferable to use Trusted Endpoints for data transfer.
+
+NOAA RDHPCS Globus Endpoints are either ''trusted'' or ''untrusted''.
+
+* All RDHPCS systems provide DTN's
+* DTNs have full access to the back-end file systems. 
+* DTNs only accept connections from pre-authorized sites. If your site can’t access the DTNs and you need that capability, submit a help desk ticket. If the security team approves, your site will be pre-authorized.
```

Comment:
Will additional changes be required here for linting?  (i.e. max 80 characters per line)

Reply:
I routinely use a short line length for convenience; I'll make sure there aren't exceptions as I review.

Reply:
Outside the "scope" of this PR.  I'm approving this PR, and resolving all comments.